.gitignore

```diff
@@ -132,3 +132,6 @@ tmp_cache
 
 # python-language-server / Rope
 .ropeproject
+
+# Generated test files
```

Comment:
Please remove this.

Reply:
Is there a reason to not have them ignored?

Reply:
I don't know what it is.

Reply:
When I run tox it is creating a folder `homeassistant-2022.9.0.dev0` with thousands of unignored files. It looks like its building a tar file during the build process, and this folder is the staging area. Anyways, when the thousands of files are dropped on the FS it crashes VS Code's git integration. I've found adding `*.dev0/` this folder prevents the crash.

Reply:
Ah. So `dev0` here is just the version number of dev, can't be used as a matcher. I don't run `tox` myself but always use `pytest tests/components/awair`. Let me check on this.

Reply:
Removed the change for now, I will see if there is a way I can exclude it locally

Reply:
Checked with some others and the conclusion was that no one actually uses Tox 😅 We're going to remove it and rewrite the dev docs https://github.com/home-assistant/core/pull/76582